snap/snapcraft.yaml

```diff
@@ -29,8 +29,39 @@ apps:
       - network-bind
 
 parts:
+
+  lensfun:
+    plugin: cmake
+    source: https://github.com/lensfun/lensfun.git
+    source-commit: v0.3.2
+    configflags:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/usr
+      - -DBINARY_PACKAGE_BUILD=On
+
+    build-packages:
+      - python3
+      - libglib2.0-dev
+      - libpng-dev
+      - doxygen
+      - python3-docutils
+
+    override-build: |
+      snapcraftctl build
+      # run lensfun-update-data and update lens data
+      PYTHONPATH=/usr/lib/python3.6/site-packages/ lensfun-update-data
+      mv -v /var/lib/lensfun-updates/version_1/*.xml ${SNAPCRAFT_PART_INSTALL}/usr/share/lensfun/version_1
```

Comment:
So `lensfun-update-data` fetches things into `/var/lib/lensfun-updates/`? Just trying to understand what's happening here.

Reply:
Yes - you could instead map `/var/lib/lensfun-updates` into $SNAPCRAFT_USER_DATA (or whatever its called) and run it within the installed snap context.  I didn't want to change the dynamics of how the snap is currently working - so I just splattedit on top

Reply:
Very good, makes sense thank you.